styleguide/derek/pattern-components/headers/_headers.scss

```diff
@@ -149,3 +158,86 @@
     width: 70%;
   }
 }
+
+
+
+.refreshBanner {
+  @include make-banner;
+  background-image: url('https://0fb555c65f288d09a992-a2085a232a1d2a458a641f48ff12bf08.ssl.cf2.rackcdn.com/LP-assets/BG.png');
```

Comment:
same here

Reply:
This image took a second to load for me in my browser. I just looked and this file is 1.7MB.  Can this be converted to a JPG?